ssl-config/src/main/java/com/palantir/trust/TrustStores.java

```diff
@@ -62,29 +55,17 @@ public static SSLSocketFactory createSslSocketFactory(
             Path trustStorePath,
             Optional<String> trustStoreType,
             Optional<String> trustStorePassword) {
-        try {
-            TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(
-                    TrustManagerFactory.getDefaultAlgorithm());
-            KeyStore keyStore = KeyStore.getInstance(trustStoreType.or("JKS"));
-            try (InputStream stream = Files.newInputStream(trustStorePath)) {
-                keyStore.load(stream, trustStorePassword.transform(TO_CHAR_ARRAY).orNull());
-            }
 
-            trustManagerFactory.init(keyStore);
+        com.palantir.remoting.ssl.TrustStoreConfiguration trustStoreConfiguration =
```

Comment:
Can't -- this is the old deprecated file calling into the new one.

There's the deprecated `com.palantir.trust.TrustStoreConfiguration` and the new `com.palantir.remoting.ssl.TrustStoreConfiguration`. Can only import one, so the other has to be fully qualified.